exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/join/JoinUtils.java

```diff
@@ -65,6 +68,11 @@
   }
   private static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(JoinUtils.class);
 
+  public static final String FAILED_TO_PLAN_CARTESIAN_JOIN = String.format(
+      "This query cannot be planned possibly due to either a cartesian join or an inequality join. %n" +
```

Comment:
This should be `\n`.

Reply:
Hi, %n was used intentionally because it's platform independent. Please check the answer https://stackoverflow.com/questions/7833689/java-string-new-line/38077906#38077906 .

Reply:
Hi @ihuzenko , have you tested on Sqlline ? (not just through unit tests).  On my MacBook this is how it displays (all one line): 
Error: UNSUPPORTED_OPERATION ERROR: This query cannot be planned possibly due to either a cartesian join or an inequality join. If a cartesian or inequality join is used intentionally, set the option 'planner.enable_nljoin_for_scalar_only' to false and try again.

BTW, can you check the other comment I had provided..we would need to fix that issue.